fog/distribution/src/config.rs

```diff
@@ -73,10 +73,6 @@ pub struct Config {
     #[clap(long, default_value = "0", env = "MC_ADD_TX_DELAY_MS")]
     pub add_tx_delay_ms: u64,
 
-    /// Block version to use (otherwise fall back to ledger)
-    #[clap(long, env = "MC_BLOCK_VERSION")]
-    pub block_version: Option<u32>,
```

Comment:
This still seems useful for trying older block versions

Reply:
maybe, but this isn't a very convenient tool for that kind of testing